src/inmanta/server/services/compilerservice.py

```diff
@@ -623,7 +623,6 @@ async def request_recompile(
         metadata: Optional[JsonType] = None,
         env_vars: Optional[Mapping[str, str]] = None,
         partial: bool = False,
-        removed_resource_sets: Optional[list[str]] = None,
```

Comment:
I wonder if we should keep this in first instance to make the migration smoother (i.e. split this MR in two parts). 
(not a hard requirement, I just wonder)

Reply:
I don't have a good feel for how much pain this might cause, so I'll keep the parameter for now but simply ignore it.
Follow-up PR that removes it [here](https://github.com/inmanta/inmanta-core/pull/7730)